packages/office-addin-dev-certs/src/uninstall.ts

```diff
@@ -18,34 +18,49 @@ function getUninstallCommand(machine: boolean = false): string {
          throw new Error(`Platform not supported: ${process.platform}`);
    }
 }
-
+ 
 // Deletes the generated certificate files and delete the certificate directory if its empty
 export function deleteCertificateFiles(certificateDirectory: string = defaults.certificateDirectory): void {
    if (fsExtra.existsSync(certificateDirectory)) {
       fsExtra.removeSync(path.join(certificateDirectory, defaults.localhostCertificateFileName));
       fsExtra.removeSync(path.join(certificateDirectory, defaults.localhostKeyFileName));
       fsExtra.removeSync(path.join(certificateDirectory, defaults.caCertificateFileName));
-
+ 
       if (fsExtra.readdirSync(certificateDirectory).length === 0) {
          fsExtra.removeSync(certificateDirectory);
       }
    }
 }
-
+ 
 export async function uninstallCaCertificate(machine: boolean = false, verbose: boolean = true) {
    if (isCaCertificateInstalled()) {
       const command = getUninstallCommand(machine);
-
+ 
       try {
-         console.log(`Uninstalling CA certificate "Developer CA for Microsoft Office Add-ins"...`);
-         execSync(command, {stdio : "pipe" });
+         console.log(`Uninstalling CA certificate "Developer CA for Microsoft Office Add-ins"...`);
+         execSync(command, { stdio: "pipe" });
          console.log(`You no longer have trusted access to https://localhost.`);
       } catch (error) {
          throw new Error(`Unable to uninstall the CA certificate.\n${error.stderr.toString()}`);
       }
    } else {
       if (verbose) {
-         console.log(`The CA certificate is not installed.`);
+         console.log(`The CA certificate is not installed.`);
       }
    }
 }
+ 
+// Currently this method is only intended to be used for Mac due to problems on Mac that occur when CA certificates expire
+export async function uninstallExpiredCaCertificate() {
```

Comment:
don't export this

Reply:
actually I don't think you need this function at all. The uninstall command is the same and the message doesn't need to say whether it is expired. can't you just use the regular uninstallCaCertificate() function?

Reply:
I can't use the regular uninstallCaCertificate because it calls isCaCertificateInstalled, which will then get us in an endless loop.

Reply:
That's why I initially had uninstallCaCertificate take a Boolean expiredCert param so we could bypass the call to isCaCertificateInstalled for the removing of an expired cert

Reply:
Would it help to add an optional param "expiredOnly = false" to uninstallCaCertificate() which could be called with expiredOnly as true for Mac?